indigo/bulk_creator.py

```diff
@@ -199,13 +207,17 @@ def get_datatable_gsheets(self, spreadsheet_id, sheet_name):
                 .execute()
         except HttpError as e:
             self.log.warning("Error getting data from google sheets for {}".format(spreadsheet_id), exc_info=e)
-            raise ValidationError("Unable to access spreadsheet. Is the URL correct and have you shared it with {}?".format(
-                self._gsheets_secret['client_email'],
-            ))
+            raise ValidationError(
+                _("Unable to access spreadsheet. Is the URL correct and have you shared it with %s?").format(
+                    self._gsheets_secret['client_email'],
+                ))
 
         rows = result.get('values', [])
         if not rows or not rows[0]:
-            raise ValidationError("There doesn't appear to be data in sheet {} of {}".format(sheet_name, spreadsheet_id))
+            raise ValidationError(_("There doesn't appear to be data in sheet %(name)s of %(id)s") % {
+                'name': sheet_name,
+                'id': spreadsheet_id
+            })
```

Comment:
```suggestion
            }))
```

Reply:
It's currently correct: `raise ValidationError(_("...") % { ... })`.

The interpolation happens after the `_(...)` translation.